src/backends/ilp-quoter/index.js

```diff
@@ -30,9 +30,13 @@ class ILPQuoter {
 
   * putPair (uri) {
     const req = request({ method: 'PUT', uri, json: true })
-    const res = yield req
-    if (res.statusCode >= 400) {
-      throw new UnsupportedPairError(req.uri)
+    try {
+      const res = yield req
+      if (res.statusCode >= 400) {
```

Comment:
The point of this fix is not to throw though -- otherwise the call to `* connect ()` will fail without having enabled all the pairs. Please refer to original ticket.